gax/test/showcase-echo-client/package.json

```diff
@@ -28,16 +28,16 @@
     "clean": "gts clean",
     "compile": "tsc -p . && cp -r protos build/ && minifyProtoJson",
     "compile-protos": "compileProtos src",
-    "prefetch": "rm -rf node_modules package-lock.json google-gax*.tgz gapic-tools*.tgz && cd ../.. && npm pack && mv google-gax*.tgz test/showcase-echo-client/google-gax.tgz && cd ../tools && npm install && npm pack && mv gapic-tools*.tgz ../gax/test/showcase-echo-client/gapic-tools.tgz",
-    "prepare": "npm run compile-protos && npm run compile"
+    "prepare": "npm run compile-protos && npm run compile",
+    "prefetch": "rm -rf node_modules package-lock.json google-gax*.tgz gapic-tools*.tgz && cd ../.. && npm pack && mv google-gax*.tgz test/showcase-echo-client/google-gax.tgz && cd ../tools && npm install && npm pack && mv gapic-tools*.tgz ../gax/test/showcase-echo-client/gapic-tools.tgz"
   },
   "dependencies": {
     "google-gax": "./google-gax.tgz"
   },
   "devDependencies": {
-    "@types/node": "^16.0.0",
+    "@types/node": "^20.11.26",
     "gapic-tools": "./gapic-tools.tgz",
-    "typescript": "^4.5.5"
+    "typescript": "5.1.6"
```

Comment:
Why pin?

Reply:
I can revert if we want! it got generated this way